src/content/structured/components/text-input/guidance.mdx

```diff
@@ -87,6 +87,8 @@ Use helper text in addition to the field label when additional information helps
   />
 </DoubleDoDontCaution>
 
+When setting min and max values using the min and max props, make sure to add an indication of these formatting rules in the helper text.
```

Comment:
"When setting minimum and maximum values..." and can we use the code styling for 'min' and 'max' props?